### PR TITLE
Feature/link image upgrade

### PR DESCRIPTION
**Pull request by nlkluth**
### What
Upgrade `<Link />` `<Image />` and fonts

### Why
Continuing migration to latest Next features

### Resources
See reference doc https://nextjs.org/docs/app/building-your-application/upgrading/app-router-migration#upgrading-new-features

### CODE REVIEW

File: packages/nextjs/app/migration/about.tsx [hunk 1]
```diff
@@ -60,14 +60,19 @@ const AboutPage: NextPage = () => {
         </div>
         <div className="order-2">
           <NextImage
+            alt="big picture diagram"
```
Comment (colinrobertbrooks) [inline packages/nextjs/app/migration/about.tsx:63]: ❤️

File: packages/nextjs/app/migration/about.tsx [hunk 2]
```diff
@@ -60,14 +60,19 @@ const AboutPage: NextPage = () => {
...
             src={require("../../../../docs/img/big-picture.png")}
             loader={localImageLoader}
-            layout="intrinsic"
             width={DIAGRAM_WIDTH}
             height={DIAGRAM_HEIGHT}
-            objectFit="contain"
-            objectPosition="center"
             className="rounded-2xl"
+            priority
+            style={{
```
Comment (colinrobertbrooks) [inline packages/nextjs/app/migration/about.tsx:70]: Can we do these with Tailwind instead of style objects?

**Review by colinrobertbrooks — APPROVED** [review]
Nice! One question about styling.

File: packages/nextjs/app/migration/about.tsx (hunk 2 above)
Comment (nlkluth) [reply packages/nextjs/app/migration/about.tsx]: Sure can! 👍

Comment (colinrobertbrooks) [reply packages/nextjs/app/migration/about.tsx]: Nice! Looks great.

**Pull request merged**